INN/umbrella-citylimits
# SCAMP ad widget styles

## Changes

This pull request makes the following changes:

- [x] adds some basic styles for loaded DFW widgets on homepage widget area, header widget area, and article
    - beige background
    - vertical padding based on column width, to prevent large blank spaces on mobile devices
- [ ] does not yet implement the bracket-border-before and bracket-border-after mixins on article ads; the bracket-border-before mixin would conflict with SCAMP's `::before` to add the "advertisement" label above the ad unit.

## Why



For the homepage ads in #44 and the article ads in #55

## Testing/Questions

Features that this PR affects:

- SCAMP ad widgets


Questions that need to be answered before merging:

- [ ] Do we want to merge this and deploy it to staging based solely on code review? We don't have enough config info from staging to be sure that this will work.

Steps to test this PR, until we get a working ad code from City Limits:

1. Install and activate https://wordpress.org/plugins/doubleclick-for-wp/ and https://wordpress.org/plugins/super-cool-ad-inserter/
2. Configure SCAMP — the plugin should already be configured from recent DB dumps
3. Configure SCAIP — defaults are fine
4. Add some widgets to the Header Ad Widget, Homepage Bottom, and SCAIP ad units
5. ⚠️  hack the loaded page's ad units to remove the `display-none` class from the ad unit divs
6. ⚠️ imagine, or edit the HTML, to add an image or other creative inside the ad unit div

```html
<aside class="scaip scaip-1    ">
	<aside id="doubleclick_widget-11" class="widget widget-1 odd default widget_doubleclick_widget clearfix">
		<div class="dfw-unit dfw-lazy-load dfw-loaded display-block" data-adunit="in_story_1" data-size-mapping="mapping2" id="in_story_1-auto-gen-id-2" style="">
			<div id="google_ads_iframe_/network_code/in_story_1_0__container__" style="border: 0pt none;">
				<iframe id="google_ads_iframe_/network_code/in_story_1_0" title="3rd party ad content" name="google_ads_iframe_/network_code/in_story_1_0" width="300" height="250" scrolling="no" marginwidth="0" marginheight="0" frameborder="0" srcdoc="" data-google-container-id="3" style="border: 0px; vertical-align: bottom;" data-load-complete="true">
				</iframe>
			</div>
		</div>
	</aside>
</aside>

```
